packages/nimble-components/src/table-column/base/tests/table-cell-view.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { fixture, Fixture } from '../../../utilities/tests/fixture';
+import type { TableCellView } from '../cell-view';
+import type { DelegatedEventEventDetails } from '../types';
+import {
+    tableColumnEmptyCellViewTag,
+    TableColumnDelegatesClickAndKeydown,
+    TableColumnEmpty
+} from './table-column.fixtures';
+
+async function setup(): Promise<Fixture<TableCellView>> {
+    return fixture(tableColumnEmptyCellViewTag);
+}
+
+describe('TableCellView', () => {
+    let element: TableCellView;
+    let connect: () => Promise<void>;
+    let disconnect: () => Promise<void>;
+
+    beforeEach(async () => {
+        ({ element, connect, disconnect } = await setup());
+    });
+
+    afterEach(async () => {
+        await disconnect();
+    });
+
+    it('delegates event(s) configured by assigned column', async () => {
```

Comment:
I think it would be good to also test that a non-delegated event does not get delegated.

Reply:
I'm basically testing that already when I switch the column to one that doesn't delegate any events, but I can also test the same for the column that does delegate click and keydown.